externalPlugins/EX-PO007-NamingConventions.js

```diff
@@ -66,9 +66,17 @@ var plugin = {
 var onPolicy = function(policy, cb) {
   var displayName = policy.getDisplayName(),
```

Comment:
This example plugin should not examine the display name, but rather the name.  If we really want the example to check the display name, then the plugin file name should be changed to reflect that purpose.